kernel/kernel-defaults/src/test/scala/io/delta/kernel/defaults/LogReplayEngineMetricsSuite.scala

```diff
@@ -80,49 +66,35 @@ class LogReplayEngineMetricsSuite extends AnyFunSuite with TestUtils {
 
     val engine = new MetricsEngine(hadoopFileIO)
 
-    withTempDir { dir => f(dir.getAbsolutePath, engine) }
+    withTempDir { dir =>
+      f(dir.getAbsolutePath, engine)
+    }
   }
 
-  private def loadPandMCheckMetrics(
-      snapshotFetchCall: => StructType,
+  protected def loadPandMCheckMetrics(
+      table: Table,
       engine: MetricsEngine,
       expJsonVersionsRead: Seq[Long],
       expParquetVersionsRead: Seq[Long],
       expParquetReadSetSizes: Seq[Long] = null,
-      expChecksumReadSet: Seq[Long] = null): Unit = {
+      expChecksumReadSet: Seq[Long] = null,
```

Comment:
Not now, but if this list grows (e.g. listFrom calls), we can look at adding a case object for this and utilities to create.